lib/Configuration.php

```diff
@@ -107,7 +107,9 @@ public static function getVersion() {
 
 		$headFile = '.git/HEAD';
 
-		if(file_exists($headFile)) {
+		$is_readable = @is_readable($headFile);
+
+		if($is_readable && file_exists($headFile)) {
```

Comment:
Checking for `is_readable` makes sense, good point :+1: 

However, it doesn't make sense to use `is_readable` and `file_exists` on the same subject, because `is_readable` already checks if the file exists:

> is_readable — Tells whether a file exists and is readable
>
> -- http://php.net/manual/en/function.is-readable.php